src/store/reducers/ormReducer/index.js

```diff
@@ -588,3 +639,19 @@ export default function ormReducer (state = orm.getEmptyState(), action) {
 
   return session.state
 }
+
+// XXX: this is ugly, would be better to load these posts through redux-orm "queries" so they update automatically
```

Comment:
Not action for now--just a note: 

This is a tangle of things, but zoomed-out a bit I think it would be better mutate and rely upon the return from that mutation to make the update vs doing things optimistically. Adding/preferring a brief loading indicator until the result of the mutation comes back and updates through the Model Extractor... The amount of futzing required to do optimistic updates right now I think is creating debt which will make a transition away from our particular use of ReduxORM and such more work.

Reply:
the only issue is that not doing optimistically makes the app feel slow compared to other chat apps. right now we dont add new chats optimistically but i think people will complain about it